packages/forms/frontend/sirius-components-forms/src/form/FormEventFragments.types.ts

```diff
@@ -57,20 +57,12 @@ export interface GQLFormRefreshedEventPayload
   form: GQLForm;
 }
 
-export interface GQLRepresentationMetadata {
-  id: string;
-  label: string;
-  kind: string;
-  description: GQLRepresentationDescription;
-}
-
 export interface GQLRepresentationDescription {
   id: string;
 }
 
-export interface GQLForm {
+export interface GQLForm extends GQLRepresentation {
```

Comment:
GQLRepresentation should probably be removed entirely because it creates a wrong type. The subscription never fetches Form#metadata#id for example, only Form#metadata#label (Is that even necessary?). We won't fetch the same fields between Form, ParChart and BarChart.